django-verdant/rca/context_processors.py

```diff
@@ -9,6 +9,7 @@
     SUSTAINRCA_CATEGORY_CHOICES
 )
 from .reachout_choices import REACHOUT_PROJECT_CHOICES, REACHOUT_PARTICIPANTS_CHOICES, REACHOUT_THEMES_CHOICES, REACHOUT_PARTNERSHIPS_CHOICES
+from rca.navigation import Navigation
 
 
 def global_vars(request):
```

Comment:
See comments below about including in Wagtail admin

Reply:
Turns out my check was checking for starswith `admin/' and not `/admin/`. I've corrected it by adding the first `/` to the check in context_processors. The admin (locally) loads almost instantly now so is no longer fetching the nav when requesting admin pages.